bundles/org.openhab.binding.volumio/src/main/java/org/openhab/binding/volumio/internal/VolumioHandler.java

```diff
@@ -257,6 +307,16 @@ private void bindDefaultListener() {
      */
     @Override
     public void initialize() {
+        VolumioService volumioLocal = volumio;
+
+        if (volumioLocal == null) {
```

Comment:
Why did you add this block? 
If i'm not mistaken, volumioLocal will always be null, as this method is only called when the class initializes and therefore will allways be null.

Reply:
solved